cg/apps/hermes/hermes_api.py

```diff
@@ -27,6 +30,10 @@ def convert_deliverables(
         """Convert deliverables file in raw workflow format to CG format with Hermes."""
         LOG.info("Converting workflow deliverables to CG deliverables")
         convert_command = [
+            "run",
+            "--bind",
+            self.container_mount_volume,
+            "/home/proj/stage/singularity_containers/hermes_latest.sif",
```

Comment:
All containers are located here - even the ones used in prod. This should change in the future.